homu/parse_issue_comment.py

```diff
@@ -97,7 +97,8 @@ def hook(cls, hook_name, hook_extra=None):
     'rollup-': 0,
     'rollup': 1,
     'rollup=maybe': 0,
-    'rollup=never': -1,
+    'rollup=never': -2,
+    'rollup=iffy': -1,
```

Comment:
Can you import `WORDS_TO_ROLLUP` from `src/main.py`?

Reply:
No, because of cyclic imports, but I did the reverse